### PR TITLE
feat(stories): add story for AlertLink

### PR DESCRIPTION
**Pull request by michellewzhang**
also fixed some wording in the alert story

### CODE REVIEW

File: static/app/components/alertLink.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        </AlertLink>
+        <p>
+          Lastly, you can get creative with the <JSXProperty name="onClick" value /> prop.
+        </p>
+        <AlertLink
+          priority="info"
+          size="normal"
+          href="https://sentry.io/for/session-replay/"
+          openInNewTab
+          onClick={() => setCount(count + 1)}
```
Comment (ryan953) [inline static/app/components/alertLink.stories.tsx:147]: Is it possible to have an `onClick` without the `href`?

If yes, then i'd just do `onClick` in the example, so we can see the counter incrementing.

```suggestion
          onClick={() => setCount(count + 1)}
```

Comment (michellewzhang) [reply static/app/components/alertLink.stories.tsx]: yup just updated

**Review by ryan953 — APPROVED** [review]
I never realized the differences between Alert and AlertLink before. Different borders. hrmm.

SCR-20231222-kids

**Comment by ryan953** [comment]
To be clear we do not need to update components at the same time we're writing stories. Story changes could spark a design discussion which would be a nice and practical outcome.

**Pull request merged**